pkg/walker.go

```diff
@@ -80,6 +82,17 @@ func (w *Walker) doWalk(ctx context.Context, p *Parser, dir string, visitor Walk
 
 		if file.IsDir() {
 			if w.Recursive {
+				nextFiles, err := ioutil.ReadDir(path)
+				if err != nil {
+					log.Err(err).Msgf("Error reading directory")
+					return
```

Comment:
Because the doWalk function doesn't provide a way to bubble the error up, I feel like this should be a fatal error. But, we really should fix `doWalk` not returning errors.